quickstart.malloynb

```diff
@@ -111,11 +111,11 @@ query: table('duckdb:data/airports.parquet') -> {
   limit: 10
 }
 >>>markdown
-The basic types of Malloy expressions are `string`, `number`, `boolean`, `date`, and `timestamp`.
+The basic types of Malloy expressions are `string`, `number`, `boolean`, `date`, `timestamp`.
 >>>markdown
 ## Sources: the Basic Structure for Modeling and Reuse
 
-One of the main benefits of Malloy is the ability to save common calculations into a data model. The data model is made of *sources*, which can be thought of as tables or views, but with additional information, such as joins, dimensions and measures.
+One of the main benefits of Malloy is the ability to save common calculations into a data model. The data model is made of *sources*, which can be thought of as tables or views, but with additional information such as joins, dimensions and measures encoded into it. You can think of a *source* as a more composable SQL `WITH`.
```

Comment:
the `WITH` comparison seemed apt. Please confirm

Reply:
```suggestion
One of the main benefits of Malloy is the ability to save common calculations into a data model. The data model is made of *sources*, which can be thought of as tables or views, but with additional information such as joins, dimensions and measures encoded into it.
```

I think I prefer the comparison to views/tables, since by nature, a CTE or `WITH` statement is ephemeral and scoped to a single query; whereas views and tables are persistent objects.